spec/components/selection/Switch.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import { mount, shallow } from 'enzyme';
+import * as React from 'react';
+import { Switch } from '../../../src/components';
+import SelectionTypes from '../../../src/components/selection/SelectionTypes';
+import { SelectionInput } from '../../../src/components/selection/SelectionInput';
+import SelectionStatus from '../../../src/components/selection/SelectionStatus';
+
+describe('Switch Component', () => {
```

Comment:
Thanks for the tests!!
Even if Enzyme exists we use from now on only Testing Library

Of course don't rewrite them but it's a note for next time

Reply:
@antoinerollindev new tests are all written in Testing library could you please use it to test this component
Thank you !